### ISSUE TITLE
Error gettc initialize

### ISSUE THREAD

**Issue by ferminhg** [seri/gettc#30]
After instaling gettc in ubuntu 18.01 i got this error 

```
➜  topcoder$ gettc
Traceback (most recent call last):
	8: from /usr/local/bin/gettc:23:in `<main>'
	7: from /usr/local/bin/gettc:23:in `load'
	6: from /var/lib/gems/2.5.0/gems/gettc-2.0/bin/gettc:81:in `<top (required)>'
	5: from /var/lib/gems/2.5.0/gems/gettc-2.0/bin/gettc:36:in `init'
	4: from /var/lib/gems/2.5.0/gems/gettc-2.0/bin/gettc:36:in `new'
	3: from /var/lib/gems/2.5.0/gems/gettc-2.0/lib/gettc/download.rb:89:in `initialize'
	2: from /var/lib/gems/2.5.0/gems/gettc-2.0/lib/gettc/download.rb:206:in `get_cookie'
	1: from /var/lib/gems/2.5.0/gems/gettc-2.0/lib/gettc/download.rb:206:in `new'
/var/lib/gems/2.5.0/gems/gettc-2.0/lib/gettc/download.rb:206:in `initialize': wrong number of arguments (given 2, expected 0..1) (ArgumentError)
➜  topcoder 

```

**Comment by seri** [comment]
@ferminhg haven't got much time to work on gettc. I've just fixed this issue in gettc 2.1, please try again and let me know if it works for you

**Comment by seri** [comment]
resolved on latest version, closing
